### PR TITLE
Use wait-interpolation-smooth for objects not to run away from gripper

### PR DESCRIPTION
**Pull request by pazeshun**
I'll check if this is effective for rawlings_baseball, easter_turtle_sippy_cup, kleenex_tissue_box, soft_white_lightbulb, folgers_classic_roast_coffee and up_glucose_bottle. They easily slide too far when gripper push them.

### CODE REVIEW

**Comment by ban-masa** [comment]
+1

**Comment by pazeshun** [comment]
This PR needs jsk-ros-pkg/jsk_pr2eus#236

**Comment by pazeshun** [comment]
Tested in real robot with jsk-ros-pkg/jsk_pr2eus#236

**Comment by k-okada** [comment]
the PR is merged and released.

--
◉ Kei Okada

**Comment by pazeshun** [comment]
@k-okada Thank you!
rebase origin/master and add 2c67298 to use jsk-ros-pkg/jsk_pr2eus#236

**Comment by pazeshun** [comment]
@wkentaro is 2c67298 correct approach? If you think so, please merge this.

File: fc.rosinstall [hunk 1]
```diff
@@ -39,6 +39,10 @@
     uri: https://github.com/jsk-ros-pkg/jsk_robot.git
     version: master
 - git:
+    local-name: jsk-ros-pkg/jsk_pr2eus
+    uri: https://github.com/jsk-ros-pkg/jsk_pr2eus.git
+    version: master
```
Comment (wkentaro) [inline fc.rosinstall]:
```
version: 0.3.4
```

**Comment by pazeshun** [comment]
Fixed

**Pull request merged**